### ISSUE TITLE
Grid scrolls up after open details

### ISSUE THREAD

**Issue by voltor** [vaadin/framework#11659]
- Vaadin Framework version: (  8.8.5 )
- Browser version: Google Chrome 75.0.3770.142 AND Firefox 67.0.1
- Description of the bug
```
Grid scrolls up after srolling down a grid and opening few details of grid rows
```
- Minimal reproducible example
```
1) Create a example project by the tutorial https://vaadin.com/docs/v8/framework/tutorial.html
2) Replace the code in the init method with the following:
@Override
    protected void init(VaadinRequest vaadinRequest) {
        final VerticalLayout layout = new VerticalLayout();
        layout.setSizeFull();
        
        Grid<String> grid = new Grid<>();
        grid.setSizeFull();
        
        List<String> items = new ArrayList<>();
        for(int index = 0; index < 400; index++ ) {
        	items.add( "Row " + index );
        }
        grid.setItems(items);
        grid.setDetailsGenerator( item -> {
        	return new Label(item + " details");
        });
    	grid.addItemClickListener(event -> {
			String item = event.getItem();
			if( item != null) {
		        grid.setDetailsVisible(item, !grid.isDetailsVisible(item));
			}
		});
    	grid.addColumn(String :: toString);
        layout.addComponent(grid);
        setContent(layout);
    } 
```
3) run application by the tutorial https://vaadin.com/docs/v8/framework/tutorial.html
4) open localhost:8080 in the Google Chrome
5) click on the Row 2
![image](https://user-images.githubusercontent.com/26440987/61368643-03df3a80-a897-11e9-8eeb-57061f09abb0.png)
6) click on the Row 50
- Expected behavior
The scroll will no action
The Row 50 will be on the screen
![image](https://user-images.githubusercontent.com/26440987/61368846-751eed80-a897-11e9-871e-ec45915b64ba.png)
- Actual behavior
Grid scrolls up 
![image](https://user-images.githubusercontent.com/26440987/61368904-91bb2580-a897-11e9-9887-1c2cdbd457c3.png)

**Comment by TatuLund** [comment]
The is major update in Vaadin 8.9.0 Grid's Escalator logic concerning especially details row handling. I would recommend to test it and tell us whether it has fixed this issue as well.

**Comment by Ansku** [comment]
Should be fixed by 8.9.0.
